Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/service/PersistedTasksService.java

```diff
@@ -51,20 +56,15 @@ public PersistedTasksService(
     }
 
     @Override
-    public Observable<Event<Tasks>> getTasksEvents() {
+    public Observable<Event<Tasks>> getTasksEvent() {
```

Comment:
We could discuss this but not in this PR then, there is other observables called with this convention and we should update all for consistency if we are changing anything.